lib/dirac.cpp

```diff
@@ -312,8 +289,6 @@ namespace quda {
   void Dirac::prefetch(QudaFieldLocation mem_space, qudaStream_t stream) const
   {
     if (gauge) gauge->prefetch(mem_space, stream);
-    if (tmp1) tmp1->prefetch(mem_space, stream);
-    if (tmp2) tmp2->prefetch(mem_space, stream);
```

Comment:
So now with the dynamic temporary fields how is the prefetch done?

Reply:
That's a good question.  I could add a prefetch (if enabled) whenever the temporary is requested.

Reply:
Having thought about this a bit more, I'm going to leave it as is.  We haven't really found the need for prefetching in production yet.  If we need to consider prefetching in the future here for temporaries, we can come back to that in the future.

Reply:
Staggered MG is a reasonably production reason, but spinor fields generally aren't beefy enough to matter --- this comment is more "for the record/awareness" than a "we need to discuss this now" type deal.

Reply:
Acknowledge and agreed